src/main/java/svc/managers/SMSManager.java

```diff
@@ -45,6 +46,9 @@
 	@Value("${stlcourts.clientURL}")
 	String clientURL;
 	
+	@Value("${stlcourts.twilio.phoneNumber}")
```

Comment:
Is this value already present in the yml as a variable that can be provided at runtime? AKA not hard-coded. We don't want to hard-code the test phone number in the yml. We should probabl configure docker cloud to provide these.